README.md

```diff
@@ -146,106 +274,151 @@ You can add this on the same HTML file you added the translatable strings before
+
 localized content, add the following template tag:
 
-```
+```django
 {% load transifex %}
 ```
 
-Translations in Django templates use one of two template tags, `{% t %}` and
-`{% ut %}`. They translate strings or variables that contain strings. The
-strings themselves can be constant, or they can contain variables which can be
-resolved either from the parameters of the template tags or the context of the
-template. The difference between the two tags will be addressed later, under
-[XML escaping](#XML-escaping).
+Translations in Django templates use one of two template tags, `{% t %}` and `{% ut %}`. They translate strings or variables that contain strings. The strings themselves can be constant, or they can contain variables which can be resolved either from the parameters of the template tags or the context of the template. The difference between the two tags will be addressed later, under [XML escaping](#XML-Escaping).
 
 ```html
 <p>{% t "This is a great sentence." %}</p>
 <h2>{% t "Welcome, {username}" username=user.name %}</h2>
 <pre>{% t snippet.code  %}</pre>
 ```
 
-##### Inline and Block syntax
+##### Internationalization in Python Code
+
+Similar to template tags, each Transifex Native function call can use the same string syntax, with any combination of the below:
+
+- Constant strings
+- Variables, as named arguments.
+- ICU Message Format to express complex localization forms, that are resolved by Transifex Native SDK.
+
+```python
+text = t("Welcome, {username}", username=user.name)
+text = t("Contact", _context="support")
+```
+
+```python
+text = t(
+    "{num, plural, "
+    "    one {There is {num} user in this team.} "
+    "    other {There are {num} users in this team.}"
+    "}",
+    num=total_users,
+)
+```
+
+```python
+text = t("""
+  {gender_of_host, select,
+    female {
+      {total_guests, plural, offset:1
+        =0 {{host} does not give a party.}
+        =1 {{host} invites {guest} to her party.}
+        =2 {{host} invites {guest} and one other person to her party.}
+        other {{host} invites {guest} and # other people to her party.}
+      }
+    }
+    male {
+      {total_guests, plural, offset:1
+        =0 {{host} does not give a party.}
+        =1 {{host} invites {guest} to his party.}
+        =2 {{host} invites {guest} and one other person to his party.}
+        other {{host} invites {guest} and # other people to his party.}
+      }
+    }
+    other {
+      {total_guests, plural, offset:1
+        =0 {{host} does not give a party.}
+        =1 {{host} invites {guest} to their party.}
+        =2 {{host} invites {guest} and one other person to their party.}
+        other {{host} invites {guest} and # other people to their party.}
+      }
+    }
+  }""",
+  gender_of_host="female",
+  total_guests=current_event.total_guests,
+  host=current_event.host.user.name,
+  guest=guest.name,
+)
+```
+
+##### Inline and Block Syntax
```

Comment:
So, the sections: 'inline and block syntax', 'plurals and other complex structures', 'Parameters', 'Saving the Outcome to a Variable', 'Applying Filters to the Source String', 'XML Escaping', 'Useful Filters' and 'Metadata' are only relevant to the templatetag. The python section should be afterwards. As of now, the functionality of the python `t` function is much simpler and thus requires less explaining.

Reply:
Done